components/invitation_modal/add_to_channels.tsx

```diff
@@ -78,7 +84,7 @@ export default function AddToChannels(props: Props) {
                     id='invite_modal.example_channel'
                     defaultMessage='e.g. {channel_name}'
                     values={{
-                        channel_name: props.currentChannelName,
+                        channel_name: sampleChannelName,
```

Comment:
Can we change the name of `sampleChannelName`? Reading confused me if this was the test code but it's the actual component so maybe just `channelName`?

Reply:
Why do we need to default this to the default channel name? Shouldn't the current channel be updated to the default channel if needed?